src/components/RefinementList.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="state.items"
+      :refine="state.refine"
+      :searchable="searchable"
+      :search-for-items="state.searchForItems"
+      :toggle-show-more="toggleShowMore"
+      :is-showing-more="state.isShowingMore"
+      :createURL="state.createURL"
+      :is-from-search="state.isFromSearch"
+      :can-refine="state.canRefine"
+      :transform-Item-To-Hit="transformItemToHit"
+    >
+      <div :class="suit('searchBox')" v-if="searchable">
+        <ais-search-input v-model="searchForFacetValues"></ais-search-input>
+      </div>
+      <ul :class="suit('list')">
+        <li
+          :class="[suit('item'), {[suit('item', 'selected')]: item.isRefined}]"
+          v-for="item in state.items"
+          :key="item.value"
+        >
+          <slot name="item" :item="item" :refine="state.refine">
```

Comment:
We need to provide `searchable`, `transformItemToHit` to the slot to allow the user to do what we do in the default render.

Reply:
you have access to searchable through the parent (like showmore), so I removed it. transformItemToHit is now part of the items itself